eng/pipelines/common/templates/runtimes/send-to-helix-step.yml

```diff
@@ -76,7 +76,7 @@ steps:
         # Arcade uses this SDK instead of trying to restore one.
         DotNetCoreSdkDir: /usr/local/dotnet
 
-  - script: $(Build.SourcesDirectory)/eng/common/msbuild.sh --ci ${{ parameters.helixProjectArguments }} /maxcpucount /bl:$(Build.SourcesDirectory)/artifacts/log/SendToHelix.binlog
+  - script: $(Build.SourcesDirectory)/eng/common/msbuild.sh --ci ${{ parameters.helixProjectArguments }} /maxcpucount /p:TargetOS=${{ parameters.osGroup }} /bl:$(Build.SourcesDirectory)/artifacts/log/SendToHelix.binlog
```

Comment:
What is the problem with using `__TargetOS` env variable which is already passed through?

Reply:
If you look at helixpublishwitharcade, it does not actually seem to propagate __TargetOS: https://github.com/dotnet/runtime/blob/6b50d5f51660748f54f636f04c488e02517b052a/src/coreclr/tests/helixpublishwitharcade.proj#L18

But maybe I should fix that in helixpublishwitharcade.proj? Or is it that way for some reason?

Reply:
I think we should fix it in `helixpublishwitharcade.proj`